Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -18,6 +19,126 @@
 ALREADY_IN_PROGRESS = "create / update / delete operation is already in progress (10102)"
 
 
+def generic_retries_request(client: demisto_client,
+                            retries_message: str,
+                            exception_message: str,
+                            prior_message: str,
+                            path: str,
+                            method: str,
+                            request_timeout: int | None = None,
+                            accept: str = 'application/json',
+                            attempts_count: int = 5,
+                            sleep_interval: int = 60,
+                            should_try_handler: Callable = None,
+                            success_handler: Callable = None,
+                            api_exception_handler: Callable = None,
+                            http_exception_handler: Callable = None):
+    """
+
+    Args:
+        client: demisto client.
+        retries_message: message to print after failure when we have more attempts.
+        exception_message: message to print when we get and exception that is not API or HTTP exception.
+        prior_message: message to print when a new retry is made.
+        path: endpoint to send request to.
+        method: HTTP method to use.
+        request_timeout: request param.
+        accept: request param.
+        attempts_count: number of total attempts made.
+        sleep_interval: sleep interval between attempts.
+        should_try_handler: a method to determine if we should send the next request.
+        success_handler: a method to run in case of successful request (according to the response status).
+        api_exception_handler: a method to run in case of api exception.
+        http_exception_handler: a method to run in case of http exception
+
+    Returns: True if the request succeeded and status in case of waiting_for_process_to_end
+
+    """
+    try:
+        for attempt in range(attempts_count - 1, -1, -1):
+            try:
+                if should_try_handler and not should_try_handler():
+                    # if the method exist and we should not try again.
+                    return True, None
+
+                # should_try_handler return True, we are trying to send request.
+                logging.info(f"{prior_message}, Attempt: {attempts_count - attempt}/{attempts_count}")
```

Comment:
```suggestion
        for attempt in range(attempts_count):
            try:
                if should_try_handler and not should_try_handler():
                    # if the method exist and we should not try again.
                    return True, None

                # should_try_handler return True, we are trying to send request.
                logging.info(f"{prior_message}, attempt: {attempts_count +1}/{attempts_count}")
```
otherwise `attempt` should be named `attempts_left`

Reply:
@eyalpalo  i prefer the option for changing the variable name, rather than changing the loop direction